content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,23 +1,33 @@
 ---
-title: 'Deploy to AWS Lambda with Serverless Framework'
-metaTitle: 'Deploy your Serverless Framework project with Prisma'
-metaDescription: 'Learn how to deploy your Prisma-backed applications to AWS Lambda with Serverless Framework'
+title: 'Deploy to AWS Lambda'
+metaTitle: 'Deploy your application using Prisma to AWS Lambda'
+metaDescription: 'Learn how to deploy your Prisma-backed applications to AWS Lambda with AWS SAM, Serverless Framework, or SST'
 tocDepth: 3
 ---
 
 <TopBlock>
 
-This guide explains how to avoid common issues when deploying a project using Prisma with [Serverless Framework](https://www.serverless.com/framework/).
+This guide explains how to avoid common issues when deploying a project using Prisma to [AWS Lambda](https://aws.amazon.com/lambda/).
 
-The Serverless Framework simplifies deployment to AWS Lambda. It provides a CLI that helps with workflow automation and AWS resource provisioning. While Prisma works well with the Serverless Framework "out of the box", there are a few improvements that can be made within your project to ensure a smooth deployment and performance. There is also additional configuration that is needed if you are using the [`serverless-webpack`](https://www.npmjs.com/package/serverless-webpack) or [`serverless-bundle`](https://www.npmjs.com/package/serverless-bundle) libraries.
+While a deployment framework is not required to deploy to AWS Lambda, this guide covers deploying with [AWS Serverless Application Model (SAM)](https://aws.amazon.com/serverless/sam/), [Serverless Framework](https://www.serverless.com/framework/), and [SST](https://sst.dev/).
+
+AWS SAM is a open-source framework from AWS that can be used in the creation of serverless applications. AWS SAM includes the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-reference.html#serverless-sam-cli) which you will use to build, test, and deploy your application.
+
+Serverless Framework provides a CLI that helps with workflow automation and AWS resource provisioning. While Prisma works well with the Serverless Framework "out of the box", there are a few improvements that can be made within your project to ensure a smooth deployment and performance. There is also additional configuration that is needed if you are using the [`serverless-webpack`](https://www.npmjs.com/package/serverless-webpack) or [`serverless-bundle`](https://www.npmjs.com/package/serverless-bundle) libraries.
+
+SST provides tools that make it easy for developers to define, test, debug, and deploy their application. Prisma works well with SST, but must be configured so that your schema is correctly packaged by SST.
```

Comment:
it seems like you'r explaining each framework in following paragraphs, so to make this more clear I suggest to use a bullet list to present this information.

```suggestion
While a deployment framework is not required to deploy to AWS Lambda, this guide covers deploying with [AWS Serverless Application Model (SAM)](https://aws.amazon.com/serverless/sam/), [Serverless Framework](https://www.serverless.com/framework/), and [SST](https://sst.dev/):
- AWS SAM is a open-source framework from AWS that can be used in the creation of serverless applications. AWS SAM includes the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-reference.html#serverless-sam-cli) which you will use to build, test, and deploy your application.
- Serverless Framework provides a CLI that helps with workflow automation and AWS resource provisioning. While Prisma works well with the Serverless Framework "out of the box", there are a few improvements that can be made within your project to ensure a smooth deployment and performance. There is also additional configuration that is needed if you are using the [`serverless-webpack`](https://www.npmjs.com/package/serverless-webpack) or [`serverless-bundle`](https://www.npmjs.com/package/serverless-bundle) libraries.
- SST provides tools that make it easy for developers to define, test, debug, and deploy their application. Prisma works well with SST, but must be configured so that your schema is correctly packaged by SST.
```